pyaedt/edb_core/edb_data/siwave_simulation_setup_data.py

```diff
@@ -246,130 +358,286 @@ def mesh_frequency(self):
     def perform_erc(self, value):
-        self.sim_setup_info.SimulationSettings.AdvancedSettings.PerformERC = value
+        edb_setup_info = self.sim_setup_info
+        edb_setup_info.SimulationSettings.AdvancedSettings.PerformERC = value
+        self._parent._edb_object = self._parent._set_edb_setup_info(edb_setup_info)
         self._parent._update_setup()
 
     @mesh_frequency.setter
     def mesh_frequency(self, value):
-        self.sim_setup_info.SimulationSettings.UseCustomSettings = True
-        self.sim_setup_info.SimulationSettings.AdvancedSettings.MeshFrequency = value
+        edb_setup_info = self.sim_setup_info
+
+        edb_setup_info.SimulationSettings.AdvancedSettings.MeshFrequency = value
+        self._parent._edb_object = self._parent._set_edb_setup_info(edb_setup_info)
         self._parent._update_setup()
 
 
-class SiwaveDCAdvancedSettings(object):
+class DCSettings(SettingsBase):
     def __init__(self, parent):
-        self._parent = parent
+        super().__init__(parent)
+        self.defaults = {
+            "compute_inductance": False,
+            "contact_radius": "0.1mm",
+            "use_dc_custom_settings": False,
+            "plot_jv": True,
+        }
+        self.dc_defaults = {
+            "dc_slider_position": [0, 1, 2],
+        }
 
     @property
-    def sim_setup_info(self):
-        """EDB internal simulation setup object."""
+    def compute_inductance(self):
+        """Whether to compute Inductance.
 
-        return self._parent._edb_sim_setup_info
+        Returns
+        -------
+        bool
+            ``True`` if inductances will be computed, ``False`` otherwise.
+        """
+
+        return self.sim_setup_info.SimulationSettings.DCSettings.ComputeInductance
+
+    @compute_inductance.setter
+    def compute_inductance(self, value):
+        edb_setup_info = self.sim_setup_info
+        edb_setup_info.SimulationSettings.DCSettings.ComputeInductance = value
+        self._parent._edb_object = self._parent._set_edb_setup_info(edb_setup_info)
+        self._parent._update_setup()
 
     @property
-    def min_void_area(self):
-        """Minimum area below which voids are ignored.
+    def contact_radius(self):
+        """Circuit element contact radius.
+
+        Returns
+        -------
+        str
+        """
+        return self.sim_setup_info.SimulationSettings.DCSettings.ContactRadius
+
+    @contact_radius.setter
+    def contact_radius(self, value):
+        edb_setup_info = self.sim_setup_info
+        edb_setup_info.SimulationSettings.DCSettings.ContactRadius = value
+        self._parent._edb_object = self._parent._set_edb_setup_info(edb_setup_info)
+        self._parent._update_setup()
+
+    @property
+    def dc_slider_position(self):
+        """DC simulation accuracy level slider position. This property only change slider position.
+
+        Options:
+        0- ``optimal speed``
+        1- ``balanced``
+        2- ``optimal accuracy``.
```

Comment:
```suggestion
        Options are:

        ``0``: Optimal speed
        ``1``: Balanced
        ``2``: Optimal accuracy
```
I assume that you enter the numeric value. It should be the code entity and not the description for it.